src/OctoshiftCLI.Tests/ado2gh/Commands/GenerateScriptCommandTests.cs

```diff
@@ -852,10 +810,9 @@ public async Task PatallelScript_Skips_Team_Project_With_No_Repos()
             };
             await _command.Invoke(args);
 
-            _scriptOutput = TrimNonExecutableLines(_scriptOutput, 35, 6);
-
             // Assert
-            _scriptOutput.Should().BeEmpty();
+            _scriptOutput.Should().BeNull();
```

Comment:
theres a typo in the test method name above

Reply:
Fixed in 2f3f608394d441f9a97d5740e04e5d0fc5c35607